cookbooks/ey-cron/recipes/custom.rb

```diff
@@ -0,0 +1,86 @@
+def delete_cron_jobs_not_in_custom(user, crons)
+  # get the existing cron jobs created by this cron recipe
+  existing_crons_command = Mixlib::ShellOut.new("grep -E -o '\# Chef Name: custom_cron_(.*)' /var/spool/cron/crontabs/#{user}")
+  existing_crons_command.run_command
+  existing_cron_names = existing_crons_command.stdout
+  existing_crons = []
+
+  # get the existing cron names without the prefix custom_cron_
+  existing_cron_names.each_line do |line|
+    existing_crons << line.chomp.gsub(/\# Chef Name: custom_cron_/, "")
+  end
+  Chef::Log.debug "current custom cron jobs #{existing_crons.inspect}"
+
+  # get the cron jobs that don't exist on the custom-cron attributes
+  deleted_crons = existing_crons - crons.map { |c| c[:name] }
+  Chef::Log.debug "deleted custom cron jobs #{deleted_crons.inspect}"
+  deleted_crons.each do |deleted_cron|
+    cron "custom_cron_#{deleted_cron}" do
+      user user
+      action :delete
+    end
+  end
+end
+
+if crontab_instance?(node)
+  if node["custom_crons"]; then
```

Comment:
Do not use then for multi-line if. (https://rubystyle.guide#no-then) 

Could this be correct to inline with cookstyle please or add an exception

Reply:
Fixed to be inline with cookstyle

From:

```
  if node["custom_crons"]; then
```

To:

```
  if node["custom_crons"]
```